src/main/resources/db/changelog/changes/02-create-users-table.yaml

```diff
@@ -36,4 +36,10 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: shipping_address
-                  type: varchar(255)
\ No newline at end of file
+                  type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: tinyint(1)
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```

Comment:
Better create a new changeSet. On a real project, you would get a checksum mismatch error